XClass/index.js

```diff
@@ -0,0 +1,51 @@
+const Plugin = module.parent.require('../Structures/Plugin');
+
+class XClass extends Plugin {
+    constructor(...args) {
+        super(...args);
+        this.log('Loading listeners');
+        this.userModalBind = this.userModal.bind(this);
+        this.userPopoutBind = this.userPopout.bind(this);
+        this.messageGroupBind = this.messageGroup.bind(this);
+        window.A.Watcher.on('messageGroup', this.messageGroupBind);
+        window.A.Watcher.on('userPopout', this.userPopoutBind);
+        window.A.Watcher.on('userModal', this.userModalBind);
+    }
+
+    static get after() { return ['AFunc'] }

```

Comment:
all lower case. it's using the internal plugin manager representation and that ones using all lower case versions